python/cudf/cudf/pandas/__init__.py

```diff
@@ -25,14 +25,16 @@ def install():
     global LOADED
     LOADED = loader is not None
 
-    if (rmm_mode := os.getenv("CUDF_PANDAS_RMM_MODE", None)) is not None:
+    rmm_mode = os.getenv("CUDF_PANDAS_RMM_MODE", "managed_pool")
+    if rmm_mode != "disable":
         # Check if a non-default memory resource is set
         current_mr = rmm.mr.get_current_device_resource()
         if not isinstance(current_mr, rmm.mr.CudaMemoryResource):
             warnings.warn(
                 f"cudf.pandas detected an already configured memory resource, ignoring 'CUDF_PANDAS_RMM_MODE'={str(rmm_mode)}",
                 UserWarning,
             )
+        enable_prefetching = False
```

Comment:
We could do this and remove the parts where we set it to True.
```suggestion
        enable_prefetching = "managed" in rmm_mode
```